modules/azure/vault/vault.tf

```diff
@@ -40,6 +40,12 @@ resource "azurerm_virtual_machine" "vault" {
     version   = var.flatcar_linux_version
   }
 
+  plan {
+    name = var.flatcar_linux_channel
+    publisher = "kinvolk"
+    product = "flatcar-container-linux-free"
+  }
```

Comment:
Why do we need this `plan` block now?

Reply:
It is related to Kinvolk providing licensed versions of Flatcar via marketplace. @whites11 can give more details

Reply:
In theory we should not need this (this is normally used on Azure to get access to paid images, but this one is free).
We got in touch with Kinvolk to get that fixed, but they didn't solve it yet.
Therefore we need to keep pushing that block in all our API requests that use the flatcar linux free image.